modules/460-log-shipper/crds/cluster-logging-config.yaml

```diff
@@ -58,11 +58,20 @@ spec:
                   default: {}
                   properties:
                     namespaceSelector:
+                      oneOf:
```

Comment:
Would we need to increase the CDR version?

Does required fields break backward compatibility?

Reply:
Adding new fields is ok. Changing existing fields is not.